src/components/related/ComparisonModal.jsx

```diff
@@ -26,22 +27,24 @@ const ComparisonModal = ({left, right, togglePop}) => {
     featureHolder[value] = ''
   })
 
-// console.log(featureHolder)
   return (
     <div>
-      <div className={styles.modal} >
-      <div className={styles.actions}>
+      <div className='modal'>
```

Comment:
It would be great to if we could get by without using css modules because this is much easier to write and to read.

Reply:
I just simply changed the name of the css file, fortunately it did not break the function.